workspace/welcome-to-xod/015-xod-cloud/patch.xodp

```diff
@@ -0,0 +1,127 @@
+{
+  "comments": [
+    {
+      "content": "In this part of the tutorial, we'll only store the data in the cloud. In one of the next sections of the tutorial, we'll retrieve the data and react to it.\n\nYou can see a `cloud-store` node here. This node store data into your private feed. The feed is a storage of your data. The name of the feed is a path, like a path in URL or your filesystem. For example, `bob/smart-home-project/room1/temperature`.\n  This node prepends username and project name automatically to your feed. That's why a bound value on the `FEED` input is \"moisture\" instead of a full path.\n\nThe program is ready, but to make it works, you have to do a few things.\n\n## Excercise\n\n1. Log in into your XOD account in the Account Pane (main menu: View -> Toggle Account Pane).\n2. Open Project Preferences (main menu: Edit -> Project Preferences) ensure that the project has a name and click on the \"Generate\" button next to the \"XOD Cloud API Key\" input.\n3. Run a simulation and change the value of a `tweak-number` node a few times.\n4. Open the [feeds](https://xod.io/feeds/) page and check out that the data is stored in the feed named `YOUR_USERNAME/welcome-to-xod/moisture`.",
+      "id": "S1Xo6Npn8",
+      "position": {
+        "units": "slots",
+        "x": 14,
+        "y": 0
+      },
+      "size": {
+        "height": 6,
+        "units": "slots",
+        "width": 7
+      }
+    },
+    {
+      "content": "# XOD Cloud Feeds\n\nThe Internet-Of-Things is mostly about collecting some data in the cloud and reacting to it.\n\nIt makes it possible to make a production process on factories safer, make fewer defects, and so on. Also, it integrates into a life of ordinary people: smart fridges, smart homes, greenhouses, wearable electronics, etc.\n\nSome of the systems have their servers to collect the data, but for the individuals, it is too complicated and expensive to have such own infrastructure. Here come services like [XOD Cloud](https://xod.io/docs/guide/getting-started-with-feeds/). You can store and retrieve any data using it.\n\nThe XOD provides a standard library called `xod-cloud/feeds` to work with cloud feeds most simply. It has nodes to store and retrieve data.\n\nYou can see all your stored data on [feeds](https://xod.io/feeds/) page on the XOD website.",
```

Comment:
# XOD Cloud Feeds

The Internet-Of-Things is mostly about collecting some data in the cloud and reacting to their changes.

It can save a production process on factories, lower the defect occurrence rate, and so on. Also, it integrates into a life of ordinary people: smart fridges, homes, greenhouses, wearable electronics, etc.

Some systems have their servers to collect the data. Still, for the individuals, it is too complicated and expensive to own such infrastructure. Here comes the [XOD Cloud](https://xod.io/docs/guide/getting-started-with-feeds/) service. You can store and retrieve any data using it.

The XOD provides a standard library called `xod-cloud/feeds` to work with cloud feeds most simply. It has nodes to store and retrieve data.

You can see all your stored data on [feeds](https://xod.io/feeds/) page on the XOD website.